user/src/pages/wettbewerb/[page].astro

```diff
@@ -0,0 +1,355 @@
+});
+
+const postsResponse = await fetch(
+  import.meta.env.PUBLIC_BACKEND_URL + "/api/posts?" + stringifiedPostQuery
+);
+
+let posts = (await postsResponse.json()).docs;
+
+posts = posts.slice(0, 6);
+
+if (posts.length < 6) {
+  posts = posts.slice(0, 3);
+}
+
+// GET WRITING TIPS
+
+const guideResponse = await fetch(
+  import.meta.env.PUBLIC_BACKEND_URL + "/api/globals/writing_tips"
+);
+
+const tips = (await guideResponse.json()).tips;
+console.log(tips);
+---
+
+<Layout title="Schreibwettbewerbe Bad Harzburg">
+  <main>
+    <section
+      class="min-h-[90vh] w-screen flex flex-row 2xl:flex-col items-center justify-center relative px-52 xl:px-36 md:px-28 sm:px-4 py-20 gap-14"
+    >
+      <div class="absolute w-full h-full z-[-1] md:hidden">
+        <img
+          class="absolute left-[35%] top-16 lg:w-28"
+          src="/images/shapes/three.svg"
+          alt="Some shapes"
+        />
+        <img
+          class="absolute left-[8%] top-[30%] lg:w-16"
+          src="/images/shapes/two1.svg"
+          alt="Some shapes"
+        />
+        <img
+          class="absolute left-[-4%] bottom-0 lg:w-28 -rotate-45"
+          src="/images/shapes/three.svg"
+          alt="Some shapes"
+        />
+        <img
+          class="absolute right-[50%] bottom-10 lg:w-28 rotate-12"
+          src="/images/shapes/one.svg"
+          alt="Some shapes"
+        />
+      </div>
+      <img
+        class="w-[36rem] h-[36rem] md:w-[24rem] md:h-[24rem] rounded-2xl shadow-2xl object-cover"
+        src={import.meta.env.PUBLIC_BACKEND_URL +
+          competitionData.image_hero.url}
+        alt={competitionData.image_hero.alt}
+      />
+      <div
+        class="w-[46rem] 2xl:w-full h-[36rem] 2xl:h-full py-4 flex flex-col justify-between 2xl:gap-8"
+      >
+        <h1
+          class="text-twine-400 font-bold font-serif small-caps text-8xl 2xl:text-center lg:text-6xl leading-[6rem]"
+        >
+          {competitionData.title}
+        </h1>
+        <h2
+          class="text-3xl lg:text-2xl font-bold small-caps text-bandicoot-400 mt-3 2xl:text-center leading-8"
+        >
+          {competitionData.sponsor_string}
+        </h2>
+        <div
+          class="flex flex-row md:flex-col justify-between 2xl:justify-evenly md:items-center gap-2"
+        >
+          <div class="md:flex md:flex-col md:items-center">
+            <h3 class="text-bandicoot-400 font-bold text-basic mt-5 mb-2">
+              Altersgruppen
+            </h3>
+            <ul class="flex flex-wrap md:justify-center gap-1">
+              {
+                competitionData.agegroups.map((agegroup: any) => (
+                  <li class="w-24 h-8 border-[1px] text-sm border-bandicoot-400 rounded-lg px-2 py-1 text-bandicoot-400 flex justify-center items-center">
+                    {agegroup.age_end
+                      ? `${agegroup.age_end}-${agegroup.age_start}`
+                      : `${agegroup.age_start}+`}{" "}
+                    Jahre
```

Comment:
Hierfür gibt es unter lib/ageGroups eine Funktion, die die Altersgruppen in einen String umwandelt. Die würde ich benutzen, weil die noch ein paar Edge-Cases abfängt.